### PR TITLE
Brewfather Support

### PR DESCRIPTION
**Pull request by corbinstreehouse**
Cleanly add support for Brewfather pushing. I tested this; seems to work pretty well so far.

### CODE REVIEW

File: data/settings.htm [hunk 1]
```diff
@@ -257,6 +257,36 @@ <h5 class="card-header">
...
+                            These settings control how TiltBridge talks to <a href="http://www.brewfather.net/">Brewfather</a>
+                            With a premium account, Brewfather supports logging from gravity sensors such
+                            as the Tilt Hydrometer. To find your API key, log into your Brewfather account and go
+                            to Settings > Custom Stream. Your Stream key will be a string of letters and numbers (not a URL).
+                        </p>
+                        <form action="/settings/update/" method="POST">
+                            <div class="form-group row">
+                                <label for="brewfatherKey" class="col-sm-2 col-form-label" data-toggle="tooltip" title="Your Brewfather Stream key">Brewfather Stream key</label>
+                                <div class="col-sm-8">
+                                    <input type="text" class="form-control" name="brewfatherKey" id="brewfatherKey" placeholder="000ad814c6af391dee8f91" :value="settings.brewfatherKey">
```
Comment (thorrak) [inline data/settings.htm]: Just to be clear, this isn't your actual brewfather key, right? ;)

**Comment by corbinstreehouse** [comment]
Nope! It is just a truncate key from the copy above.

**Comment by thorrak** [comment]
Everything looks good - I'll get this compiled & tested when I get back home! Thanks again for submitting this - this actually takes care of the last feature I was hoping to add before releasing the new "v2" build targeted for the D32 Pro hardware!

**Comment by tgangso** [comment]
Excellent work :)

**Pull request merged**